test/remote/gateways/remote_borgun_test.rb

```diff
@@ -54,6 +54,27 @@ def test_successful_authorize_and_capture
     assert_success capture
   end
 
+  def test_successful_authorize_airline_data
+    passenger_itinerary_data = {
+      'MessageNumber' => '1111111',
+      'TrDate' => '20120222',
+      'TrTime' => '151515',
+      'PassengerName' => 'Jane Doe',
+      'ServiceClassCode_1' => '100',
+      'FlightNumber_1' => '111111',
+      'TravelDate_1' => '20120222',
+      'DepartureAirport_1' => 'KEF',
+      'CarrierCode_1' => 'CC',
+      'TravelAgencyCode' => 'A7654321',
+      'TravelAgencyName' => 'Spreedly Inc',
+      'TicketNumber' => '900.123.222'
+    }
+
+    options = @options.merge(passenger_itinerary_data: passenger_itinerary_data)
+    auth = @gateway.authorize(@amount, @credit_card, options)
```

Comment:
Is there anything else that'd be worth testing there? Does it respond different if itinerary data is passed in? If so, it might be worth checking for some of it here ([like was done here](https://github.com/activemerchant/active_merchant/blob/master/test/remote/gateways/remote_decidir_test.rb#L88-L90)). ¿What think ye?

Reply:
No, as best I can tell the response is the same -- it doesn't echo any of the fields back or even acknowledge them (other than not erroring out when required fields are missing).

Reply:
Also now I'm hungry for Heavenly Buffaloes, thanks for that :P

Reply:
Hahaha. It felt nice to get a little Durham into this repo :)